index.html

```diff
@@ -69,5 +69,167 @@ <h1>
         </h1>
       </div>
     </section>
+    <section class="works">
+    <div class="multi-stories">
+     <div class="grayBg"></div>
+     <div class="txt inter-font">
+        <h2>Multi-Post Stories</h2>
+        <h2>Gain+Glory</h2>
+        <ul class="projectBTNs inter-font">
+            <li> <button>Ruby on rails</button></li>
+            <li><button>css</button></li>
+            <li><button>javascript</button></li>
+            <li><button>html</button></li>
+        </ul>
+        <button class="seeProject inter-font">See Project</button>
+     </div>
+    </div>
+    <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
```

Comment:
- [ ] I would suggest you stick as much as possible to the design, please fix the spacing and it is scrolling horizontally. See the design for reference
- [ ] As you see the `li` items are covering the whole width as well kindly fix it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/185514740-44a50c28-3bc9-46e7-87f8-6c79719ae8b6.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/185514760-eee39726-a1f7-491b-a80c-7cd31b45839e.png)

Reply:
solved